redbot/cogs/economy/economy.py

```diff
@@ -904,16 +906,19 @@ async def slotmax(self, ctx: commands.Context, bid: positive_int):
         )
 
     @economyset.command()
-    async def slottime(self, ctx: commands.Context, seconds: int):
+    async def slottime(self, ctx: commands.Context, seconds: TimedeltaConverter):
         """Set the cooldown for the slot machine.
 
         Example:
             - `[p]economyset slottime 10`
+            - `[p]economyset slottime 10m`
 
         **Arguments**
 
         - `<seconds>` The new number of seconds to wait in between uses of the slot machine. Default is 5.
+        - It is also a timedeltaconverter so `1d` is converted to 1 day
```

Comment:
This isn't really clear to the user, I would suggest sth like this:
```suggestion
        - `<seconds>` The new duration to wait in between uses of the slot machine. Default is 5 seconds.
        Accepts: seconds, minutes, hours, days, weeks
```